src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.PowerShell.Commands
     /// Cmdlet to stop computer.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Online help tests for PowerShell Cmdlets.Validate 'get-help Stop-Computer -Online'
```powershell
Expected strings to be the same, but they were different.
String lengths are both 47.
Strings differ at index 44.
Expected: '...=2097151'
But was:  '...=2097060'
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Help/HelpSystem.OnlineHelp.Tests.ps1: line 48
48:                 $actualURI | Should -Be $cmdlet.HelpURI
```